package.json

```diff
@@ -47,9 +51,14 @@
     "@babel/core": "^7.20.2",
     "@babel/eslint-parser": "^7.19.1",
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.8.1",
+    "@ember/test-helpers": "file:.yalc/@ember/test-helpers",
     "@embroider/test-setup": "^1.8.3",
     "@glimmer/component": "^1.1.2",
+    "@glimmer/interfaces": "^0.84.2",
+    "@glimmer/reference": "^0.84.2",
```

Comment:
IDK why I had to install these but I had a lot of typescript errors in node_modules without them.

Reply:
Oh, that seems *very* bad. I will check out the PR and see if I can understand why.

Reply:
For what it's worth, after removing the `yalc` bit, I don't see this. I haven't yet tried it *with* a `yalc`'d `@ember/test-helpers`, though.

Reply:
I'll try again now that everything is further along.